Apps/WebClient/test/functional/e2e/cypress/integration/timeline/filter.js

```diff
@@ -7,13 +7,15 @@ describe('Filters', () => {
                  AuthMethod.KeyCloak)
     })
 
-    it('Test1', () => {
-
+    it('Validate No Records on timeline vue', () => {
+        cy.get('[data-testid=filterTextInput]').type('xxxx');
+        cy.get('[data-testid=noTimelineEntriesText]').should('be.visible');
+        cy.get('[data-testid=monthViewToggle]').click()        
```

Comment:
it seems unnecessary to swtich to the calendar view here

Reply:
fixed